src/Orleans.Core/Configuration/Options/MultiClusterOptions.cs

```diff
@@ -42,6 +51,9 @@ public class MultiClusterOptions
         /// <summary>
         /// The time between the slow retries for DOUBTFUL activations.
         /// </summary>
-        public TimeSpan GlobalSingleInstanceRetryInterval { get; set; } = TimeSpan.FromSeconds(30);
+        public TimeSpan GlobalSingleInstanceRetryInterval { get; set; } = DEFAULT_GLOBAL_SINGLE_INSTANCE_RETRY_INTERVAL;
+        public static readonly TimeSpan DEFAULT_GLOBAL_SINGLE_INSTANCE_RETRY_INTERVAL = TimeSpan.FromSeconds(30);
+
+        public Dictionary<string, string> GossipChannels { get; set; } = new Dictionary<string, string>();
```

Comment:
GossipeChannels are used in MultiClusterGossipeFactory to create `IGossipChannel` by using `AssemblyLoader.LoadAndCreateInstance<IGossipChannel>`. Should we make GossipChannels to be Dictionary<Type,String> , and use DI to configure `IGossipChannel` in `MultiClusterGossipFactory`?

Reply:
I'm of the opinion that the gossip channels are poorly structured.  I agree we should refactor them.  IMO, such refactoring is out of scope for this PR.  I added gossip channel support in the simplest way, as I don't have the bandwidth to think through how these "should" be configured.  Basically, since their currently configured poorly, I've opted to continue to support them poorly :/

Reply:
can we add a todo item for those components which use assembly loading to create instance, since this pattern contradicts the direction we are going ? And then let's discuss the priority of those todos?